src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4066,17 +3927,12 @@ namespace winrt::TerminalApp::implementation
         else if (key == Windows::System::VirtualKey::Escape)
         {
             // User wants to discard the changes they made
-            WindowRenamerTextBox().Text(WindowName());
+            WindowRenamerTextBox().Text(WindowProperties().WindowName());
```

Comment:
do we have our own `_WindowProperties` member that we should use directly instead? Why are we going through the accessor?